### DISCUSSION TITLE
anim_to_vtk tool that supports windows OS

### DISCUSSION THREAD

**Discussion by dwyanexu419** [OpenRadioss/OpenRadioss#761] [Q&A] [answered]
Hi,I don't know if there is any anim_to_vtk tool that supports windows OS?

**Comment by OlivierWienholtz** [comment]
Hello,
The support is planned.
Need to upload the scripts.

**Comment by MarianBulla** [comment]
Hi, 
please, have a look at: https://github.com/OpenRadioss/OpenRadioss/tree/main/tools/anim_to_vtk
There is a batch file for creating the animations converter to vtk for windows.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by OlivierWienholtz** [comment, marked as answer]
Hello 
In Addition :
With Commit # 43a0f6bfb89ce76c5136c99ef21660ec188c1b34
scripts have been added &
The tools are now build & available in the win64 release downloads.

**Comment by dwyanexu419** [comment]
@OlivierWienholtz  
it helps me a lot，thank you very much.
